requirements/edx/github.txt

```diff
@@ -53,12 +53,13 @@ git+https://github.com/edx/edx-val.git@0.0.8#egg=edxval==0.0.8
 -e git+https://github.com/edx/edx-search.git@release-2015-11-17#egg=edx-search==0.1.1
 -e git+https://github.com/edx/edx-milestones.git@release-2015-11-17#egg=edx-milestones==0.1.5
 git+https://github.com/edx/edx-lint.git@v0.3.2#egg=edx_lint==0.3.2
--e git+https://github.com/edx/xblock-utils.git@213a97a50276d6a2504d8133650b2930ead357a0#egg=xblock-utils
+-e git+https://github.com/edx/xblock-utils.git@v1.0.0#egg=xblock-utils==1.0.0
```

Comment:
This version number does not match what's in `setup.py` at that `xblock-utils` tag - it's `0.1a0`. Which is correct? The versions should match.